src/main/resources/default/templates/biz/tenants/user-account.html.pasta

```diff
@@ -85,6 +85,16 @@
                         @toSpokenDate(account.getUserAccountData().getLogin().getLastSeen())
                     </t:inlineInfo>
                 </i:if>
+                <i:if test="account.getUserAccountData().getLogin().getLastLogin() != null">
+                    <t:inlineInfo labelKey="LoginData.lastLogin">
+                        @toSpokenDate(account.getUserAccountData().getLogin().getLastLogin())
+                    </t:inlineInfo>
+                </i:if>
```

Comment:
do we really need both - last login and last seen? the former isn't relevant to most admins